sample/src/main/kotlin/com/slack/circuit/sample/petdetail/PetDetail.kt

```diff
@@ -116,12 +119,16 @@ private fun renderImpl(state: PetDetailScreen.State) {
   ) { padding ->
     LazyColumn(modifier = Modifier.padding(padding)) {
       item {
-        SubcomposeAsyncImage(
+        AsyncImage(
           modifier = Modifier.fillMaxWidth(),
-          model = state.photoUrl,
+          model =
+            ImageRequest.Builder(LocalContext.current)
+              .data(state.photoUrl)
+              .placeholderMemoryCacheKey(state.photoUrlMemoryCacheKey)
```

Comment:
Neat how we're able to reuse the lower res image downloaded [here](https://github.com/slackhq/circuit/pull/28/files#diff-b6be08df46974864fe44e2b21aa9ed5f3097c502e68e0e5e89c85d0178804a6bR264) as the placeholder